OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -98,6 +98,12 @@ public class BuildingCategories
 		[Desc("Delay (in ticks) until rechecking for new BaseProviders.")]
 		public readonly int CheckForNewBasesDelay = 1500;
 
+		[Desc("Delay (in ticks) before deciding which enemy to attack first.")]
+		public readonly int ChooseInitialEnemyDelay = 750;
```

Comment:
Can you elaborate a bit why you added this?
I mean, couldn't we just use `UpdateAggroInterva` only instead?